sonar-plugin-api/src/main/java/org/sonar/api/measures/MeasuresFilters.java

```diff
@@ -232,6 +236,10 @@ public M filter(Collection<Measure> measures) {
     }
   }
 
+  /**
+   * @deprecated since 5.2. Useless by design because of Compute Engine
+   */
+  @Deprecated
   private static class RuleFilter extends AbstractRuleMeasureFilter<RuleMeasure> {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1133)